### PR TITLE
New package: testssl.sh

### LINKED ISSUE #11699: [Package]: testssl.sh
### Package description

`testssl.sh` is a free command line tool which checks a server's service on any port for the support of TLS/SSL ciphers, protocols as well as some cryptographic flaws.

Among all other scripts doing this job, it might be the best one.

https://pkgs.org/download/testssl.sh

### Home page URL

https://github.com/drwetter/testssl.sh

### Source code URL

https://github.com/drwetter/testssl.sh

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ifurther**
Closes  #11699

### CODE REVIEW

**Review by 2096779623 — CHANGES_REQUESTED** [review]
Please add `TERMUX_PKG_PLATFORM_INDEPENDENT=true` .

**Comment by ifurther** [comment]
I fixed.

**Comment by ifurther** [comment]
@sylirre I fixed.

**Comment by sylirre** [comment]
I don't see that.

still-has-issues

And that is only a part of what you need to fix. There lots of similar cases.

`$TERMUX_PREFIX` is valid under build environment, but not under Termux which uses `$PREFIX` instead. You shouldn't use this variable anyway and use full path instead.

Finally, I've asked about *patch* because such things must be reviewed before changes. `sed` is not reliable as doesn't distinguish entries which should be replaced and which should not.

**Comment by ifurther** [comment]
@sylirre I add patch file and using full path

File: packages/testssl.sh/fix-hardcodepath.patch [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
++#!@TERMUX_PREFIX@/bin/env bash
+ #
+ # vim:ts=5:sw=5:expandtab
+ # we have a spaces softtab, that ensures readability with other editors too
+@@ -99,11 +99,11 @@
+ if [[ "$SHELLOPTS" =~ xtrace ]]; then
+      if "$DEBUGTIME"; then
+           # separate debugging, doesn't mess up the screen, $DEBUGTIME determines whether we also do performance analysis
+-          exec 42>&2 2> >(tee /tmp/testssl-$$.log | sed -u 's/^.*$/now/' | date -f - +%s.%N >/tmp/testssl-$$.time)
++          exec 42>&2 2> >(tee @TERMUX_PREFIX@/tmp/testssl-$$.log | sed -u 's/^.*$/now/' | date -f - +%s.%N >/tmp/testssl-$$.time)
```
Comment (sylirre) [inline packages/testssl.sh/fix-hardcodepath.patch]: @ifurther The first part is fixed. What's up with second (`>/tmp/testssl-$$.time`)? There also numerous places for where other paths are not fixed.

**Pull request merged**